console-subscriber/src/builder.rs

```diff
@@ -149,6 +159,279 @@ impl Builder {
+            .unwrap_or_else(|| Targets::default().with_default(LevelFilter::ERROR));
+
+        let console_layer = self.build_console_layer();
+
+        tracing_subscriber::registry()
+            .with(console_layer)
+            .with(tracing_subscriber::fmt::layer().with_filter(fmt_filter))
+            .init();
+    }
+
+    /// Returns a new `tracing` [subscriber] configured with a [`TasksLayer`]
+    /// and a [filter] that enables the spans and events required by the console.
+    ///
+    /// This function spawns the console subscriber's [`Server`] in its own Tokio
+    /// runtime in a background thread.
+    ///
+    /// Unlike [`init`], this function does not set the default subscriber, allowing
+    /// additional [`Layer`]s to be added.
+    ///
+    /// [subscriber]: https://docs.rs/tracing/latest/tracing/subscriber/trait.Subscriber.html
+    /// [filter]: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.Targets.html
+    /// [`Layer`]: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/layer/trait.Layer.html
+    /// [`Server`]: crate::Server
+    ///
+    /// ## Configuration
+    ///
+    /// `console_subscriber::build` supports all of the environmental
+    /// configuration described at [`console_subscriber::init`].
+    ///
+    /// ## Differences from `init`
+    ///
+    /// Unlike [`console_subscriber::init`], this function does *not* add a
+    /// [`tracing_subscriber::fmt`] layer to the configured `Subscriber`. This means
+    /// that this function will not log spans and events based on the value of the
+    /// `RUST_LOG` environment variable. Instead, a user-provided [`fmt::Layer`] can
+    /// be added in order to customize the log format.
+    ///
+    /// You must call [`.init()`] on the final subscriber in order to [set the
+    /// subscriber as the default][default].
+    ///
+    /// ## Examples
+    ///
+    /// ```rust
+    /// use tracing_subscriber::prelude::*;
+    ///
+    /// let console_layer = console_subscriber::TasksLayer::builder()
+    ///     .with_default_env()
+    ///     .build_console_layer();
+    ///
+    /// tracing_subscriber::registry()
+    ///     .with(console_layer)
+    ///     .with(tracing_subscriber::fmt::layer())
+    /// //  .with(...)
+    ///     .init();
+    /// ```
+    /// [`.init()`]: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/util/trait.SubscriberInitExt.html
+    /// [default]: https://docs.rs/tracing/latest/tracing/dispatcher/index.html#setting-the-default-subscriber
+    /// [sub]: https://docs.rs/tracing/latest/tracing/trait.Subscriber.html
+    /// [`tracing_subscriber::fmt`]: https://docs.rs/tracing-subscriber/latest/tracing-subscriber/fmt/index.html
+    /// [`fmt::Layer`]: https://docs.rs/tracing-subscriber/latest/tracing-subscriber/fmt/struct.Layer.html
+    /// [`console_subscriber::init`]: crate::init()
+    #[must_use = "build_console_layer() without init() will not set the default tracing subscriber"]
```

Comment:
this no longer returns something that we can call `init` on, so perhaps we should change this:
```suggestion
    #[must_use = "a `Layer` must be added to a `tracing::Subscriber` in order to be used"]
```
or something like that?